package.json

```diff
@@ -19,14 +19,14 @@
     "@mycrypto/ui": "0.20.3",
     "@parity/qr-signer": "0.3.1",
     "axios": "0.18.1",
-    "bip39": "2.5.0",
+    "bip39": "3.0.2",
     "bn.js": "4.11.8",
     "bootstrap-sass": "3.3.7",
     "classnames": "2.2.5",
     "crypto-js": "3.1.9-1",
     "cryptocurrency-icons": "0.13.0",
     "electron-updater": "4.2.0",
-    "ethereumjs-abi": "git://github.com/ethereumjs/ethereumjs-abi.git#09c3c48fd3bed143df7fa8f36f6f164205e23796",
+    "ethereumjs-abi": "ethereumjs/ethereumjs-abi#09c3c48",
```

Comment:
Can we use a version of `ethereumjs-abi` on npm here?

Reply:
what do you suggest ?

Reply:
AFAIK we used a version on Git because there was some kind of bug. That was a while ago and the changes are merged (see ethereumjs/ethereumjs-abi#60), so using the latest version on npm should work.

Reply:
Great thx.
I did the change.